src/main/groovy/jobLib/rhsmLib.groovy

```diff
@@ -6,7 +6,7 @@ class rhsmLib {
     static String candlepinRepo = "candlepin"
     static String submanRepo = "subscription-manager"
     static String candlepinJobFolder = "candlepin"
-    static String submanJobFolder = "subscription-manager"
+    static String submanJobFolder = "RHSM"
```

Comment:
renamed the folder to RHSM so we can easily clean up the old stuff by deleting the current 'subscription-manager' folder

Reply:
Sounds like a good plan to me.